packages/features/bookings/components/event-meta/Members.tsx

```diff
@@ -37,9 +39,21 @@ export const EventMembers = ({ schedulingType, users, profile }: EventMembersPro
     title: `${user.name}`,
     image: "image" in user ? `${user.image}` : `/${user.username}/avatar.png`,
     alt: user.name || undefined,
-    href: user.username ? `${CAL_URL}/${user.username}` : undefined,
+    href: user.username
+      ? `${entity.orgSlug ? getOrgFullDomain(entity.orgSlug) : CAL_URL}/${user.username}`
+      : undefined,
```

Comment:
```suggestion
    href: `/${user.username}`,
```

Reply:
There is `userBookerUrl` hook to do that but I think we can just use relative URL here as this component is meant only for Booker pages

Reply:
Good point, reverted, thanks @hariombalhara